design/upf-selection-design.md

```diff
@@ -0,0 +1,32 @@
+# UPF Selection Design for Multiple DNNs
+
+## 1. Introduction
+- **Objective:** Modify the SMF to support UPF selection based on DNN and Slice ID.
+
+## 2. Discussion Points
+
+### 2.1. DNN: `internet`
```

Comment:
It would be good to provide the example of conifguration and then explain what current code can not do and what you intend to do.

Reply:
Updated version of the document that includes the example configurations and explains the current limitations as well as the intended changes